internal/pkg/runtime/engines/singularity/container.go

```diff
@@ -399,9 +404,13 @@ func (c *container) mount(point *mount.Point) error {
 			if flags&syscall.MS_REMOUNT != 0 {
 				return fmt.Errorf("can't remount %s: %s", point.Destination, err)
 			}
-			// mount error for filesystems is considered fatal
 			if point.Type != "" {
-				return fmt.Errorf("can't mount %s filesystem to %s: %s", point.Type, point.Destination, err)
+				if point.Source == "devpts" {
```

Comment:
I would go for something more generic here, like the possibility to register for a mount point an "error" function executed in case of error. But it's good for now